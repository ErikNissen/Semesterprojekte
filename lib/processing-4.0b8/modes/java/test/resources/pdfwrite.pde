version https://git-lfs.github.com/spec/v1
oid sha256:3597dd4b95e2714fe7ba08eb36357b175b1e833d9e76e6bc5fe70f65708450d4
size 217
