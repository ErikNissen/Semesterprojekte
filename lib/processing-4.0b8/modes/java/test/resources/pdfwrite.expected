version https://git-lfs.github.com/spec/v1
oid sha256:380b1edb5525fe59c729ffba6c87849b5731f76c95247e9623fb5d5dee4ed8e4
size 952
